packages/provider/src/provider.ts

```diff
@@ -59,14 +59,6 @@ export class Provider implements IProvider {
     await this.#proxy.connect()
   }
 
-  public async disconnect() {
```

Comment:
Why did you remove it?

Reply:
coz u cant disconnect the wallet extension from app, it's acquirable only through the extension